glean-core/src/upload/mod.rs

```diff
@@ -98,7 +106,12 @@ impl RateLimiter {
         }
 
         if self.count == self.max_count {
-            return RateLimiterState::Throttled;
+            let remaining = self.interval.as_millis() - self.elapsed().as_millis();
```

Comment:
Maybe worth calling out here that elapsed is always smaller than interval, because we already check that in `should_reset`?